.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: build
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Since `main` is the default branch of this repository, I guess it's better to specify it here. I'm not very familiar with GitHub Actions, is this the right way?

Reply:
OK.
I changed to the main branch.
Please check.